opsdroid/connector/websocket/__init__.py

```diff
@@ -124,8 +124,8 @@ def __init__(self, config, opsdroid=None):
             if message.target is None:
                 message.target = next(iter(self.active_connections))
             _LOGGER.debug(
-                "Responding with: '" + message.text + "' in target " + message.target
+                _("Responding with: '" + message.text + "' in target " + message.target)
```

Comment:
Would you mean replacing this strip with the other convention? Instead of using <string> + var + <string> we should keep it like the rest and use the %s convention.

If not I can change it after I merge the PR 😄